metrics/web/lib/auth/presentation/models/auth_state.dart

```diff
@@ -0,0 +1,14 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+/// Represents an auth state of user.
+enum AuthState {
+  /// Represents logged-in user state.
```

Comment:
🏷️ 
```suggestion
  /// Represents the logged in user state.
```
Please, apply this suggestion to all similar situations.